mimic/model/clb_objects.py

```diff
@@ -6,6 +6,7 @@
 from twisted.python import log
 from characteristic import attributes, Attribute
 from mimic.canned_responses.loadbalancer import (load_balancer_example,
+                                                 _prep_for_list,
```

Comment:
Does not look like `_prep_for_list` is used by anything else in `canned_responses/loadbalancers.py`